### PR TITLE
fix: view create course

### PR DESCRIPTION
**Pull request by saidabdul80**
sorted to admin  side nav and content was make to be vue component and as well backend connected

### CODE REVIEW

File: app/Http/Controllers/Api/AuthController.php [hunk 1]
```diff
@@ -35,7 +35,7 @@ public function uuid()
     public function login(Request $request)
     {
         $validator = Validator::make($request->all(), [
-            'username' => 'required',
+            'email' => 'required',
             'password' => 'required',
```
Comment (Auwalms) [inline app/Http/Controllers/Api/AuthController.php]: Again @saidabdul80

File: app/Http/Controllers/Api/AuthController.php [hunk 2]
```diff
@@ -44,7 +44,7 @@ public function login(Request $request)
         }
 
         $credentials = [
-            'username' => $request->get('username'),
+            'email' => $request->get('email'),
```
Comment (Auwalms) [inline app/Http/Controllers/Api/AuthController.php]: Here we go again

File: resources/js/components/createCourse.vue [hunk 3]
```diff
@@ -535,17 +535,40 @@
...
+								}).then((result) => {								  
+								  if (result.isConfirmed) {
+								    location.reload();
+								  } else if (result.isDenied) {								    
+								  }
+								})
+				   			}
+			            	//console.log(response);			   				           
 			            }, function(e) {		  
+			            	console.log(e.status)
```
Comment (Auwalms) [inline resources/js/components/createCourse.vue:571]: Sha no forget this here. It shouldn't make it to production

File: resources/js/plugin/plugin.js [hunk 4]
```diff
@@ -37,7 +39,81 @@ export default {
       		$('#toastdesc').html(msg);
       		x.addClass("show");
       		setTimeout(function(){x.removeClass("show"); }, 5000);
+  		},
+  		flogout(){
+  			location.href = "/logout";
```
Comment (Auwalms) [inline resources/js/plugin/plugin.js:45]: This guy, you too like play, logout Abi flogout?😂😂😂

**Review by Auwalms — CHANGES_REQUESTED** [review]
Please move the code back to username and not email address

File: resources/js/plugin/plugin.js (hunk 4 above)
Comment (saidabdul80) [reply resources/js/plugin/plugin.js:45]: frontend Logout, if at a point i try to make request and it return unAuthorize 401. then it should logout the user  from the front end without sending request to logout from backend. however there is already logout() which goes through the backend

**Pull request closed without merging**